contrib-docs/how-to-write/what-we-write.md

```diff
@@ -0,0 +1,91 @@
+---
+title: What we write
+slug: <MDN/Contribute/how-to-write/what-we-write>
+tags:
+  - <Guide>
+  - <Guidelines>
+  - Writing
+  - Documentation
+  - MDN Meta
+---
+<{{MDNSidebar}}>
+
+In this article, you'll find information about whether or not a given topic and/or type of content should be included on MDN.
+
+> **Note:** Be aware that all contributions to MDN fall under specific open source licenses; these are [described in detail](/en-US/docs/MDN/About#copyrights_and_licenses) on our [About MDN](/en-US/docs/MDN/About) page. Keep in mind that Mozilla's [Websites & Communications Terms of Use](https://www.mozilla.org/en-US/about/legal/terms/mozilla/) are in effect when you use or contribute to MDN. <!--- links need to be revisited--->
+
+## Topics that belong on MDN
+
+In general, if it's an open web-facing technology, we document it on MDN. This includes any feature that can be used by web developers creating websites and applications, now and in the near future. If a feature is implemented by multiple browsers and either accepted as standard or is progressing towards standardization, then yes, we definitely document it here. If a feature is still very experimental and not implemented in multiple browsers and/or liable to change, then it is still suitable for inclusion, but may not be seen as a priority for the writer's team to work on.
```

Comment:
I think we should mention 'specification' here - one direction I'm pretty strong about is focussing primarily on w3c specifications - there's still a few things on mdn we shouldn't be keeping, so we can discuss how to word this